test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
no cuz i dont need to be long it get a little  complicated if id is long, my opinion is doest not require to be long in demo application like this